tests/general/InvalidLinkBearTest.py

```diff
@@ -82,6 +82,7 @@ def test_run(self):
         # Quotes
         "https://github.com/coala-analyzer/coala-bears/issues/200"
         'http://httpbin.org/status/203'
+        ('http://httpbin.org/status/200').install_command()
```

Comment:
yes, previously this failed because it separated at the wrong dot - actually gmail web UI highlights this one wrong as well :P